indicators/NO_WPIT_001/R/NO_WPIT_001.qmd

```diff
@@ -0,0 +1,771 @@
+V <- lesVannforekomster(vannkategori, filsti = "../data",
+                        CACHE = "VF.RData")
+```
+
+The output informs us of several adjustments made to the data, but that the operation was successful and how many water bodies (of the relevant water category) there are.
+
+### 9.4. Calculating mEQRs using reference levels
+
+According to the WFD and DGVF ([2025](#veil2)), measurements of WFD variables are transformed into so-called nEQR values ("normalised Ecological Quality Ratios") between 0 and 1, which correspond to indicator values sensu [SEEA EA](#seeaea). 
+This is achieved in the following steps:
+
+1. *Truncation.* Variable values are truncated, where measurements that are poorer than X~0~ are set to X~0~, and measurements that are better than X~100~ are set to X~100~.
+2. *Scaling.* The truncated variable values are linearly scaled so that X~0~ becomes 0 and X~100~ becomes 1. The resulting indicator values are referred to as **EQR**s ("Ecological Quality Ratios").
+3. *Transformation.* EQR values are transformed in a stepwise-linear manner, so that the indicator values 0.2, 0.4, 0.6 and 0.8 correspond to the thresholds between the normative definitions of "bad", "poor", "moderate", "good" and "high" ecological status according to the WFD. The resulting indicator values are referred to as **nEQR**s ("normalised EQRs").
+
+Truncation, however, destroys information about variation in ecological condition that is important for estimating (modelling and extrapolating) the indicator value and its uncertainty. 
+Therefore, the dataflow from WFD to an Ecological Condition Account should be based on values that are *scaled and transformed but not truncated*.
+This is achieved by modifying the order of the steps from truncation&ndash;scaling&ndash;transformation to scaling&ndash;transformation&ndash;aggregation&ndash;truncation (following [Sandvik 2019](#san19): pp. 13--15, [Sandvik 2024](#san24)):
+
+1. *Scaling.* The *un*truncated variable values are linearly scaled so that X~0~ becomes 0 and X~100~ becomes 1. Note that the resulting values may be negative (if X is poorer than X~0~) or larger than 1 (if X is better than X~100~).
+2. *Transformation.* The scaled values are transformed in a stepwise-linear manner in accordance with the threshold values defined by the WFD. The resulting indicator values are here referred to as **mEQR**s ("modified EQRs"). This step requires an adjustment in cases were the minimum and/or maximum values of X can be much lower or higher than X~0~ or X~100~. The approach chosen here ensures that no indicator value can be below &minus;0.2 or above 1.2.
+3. *Aggregration.* Indicator values for the specific water bodies need to be spatially aggregated (see below). 
+4. *Truncation.* When mEQR values are truncated, they become identical to nEQR values. However, truncation is here done only after the spatial aggregation of the modelled and simulated values.
+
```

Comment:
If I understand this part correctly, the first part describes how the calculations are done according to WFD. Then the second part explains a problem, and suggests an alternative calculation with transformation instead of truncation. And I assume this is the approach used here. Can that be stated more clearly that this is the approach used here?